.github/workflows/test.yml

```diff
@@ -208,3 +213,21 @@ jobs:
       - name: Run the tests
         working-directory: ./myadmin
         run: npm run test
+
+  update-sandbox-repository:
+    runs-on: ubuntu-latest
+    # Only run on new tags that target a release (not latest nor next) and avoid alpha and beta tags
+    if: github.event_name == 'push' && github.ref_type == 'tag' && github.ref_name == 'refs/tags/v*' && !contains('beta', github.ref_name) && !contains('alpha', github.ref_name)
+    needs: [typecheck, simple-example-typecheck, unit-test, e2e-test]
+    steps:
+        - name: Checkout
+          uses: actions/checkout@v3
+        - name: Use Node.js LTS
+          uses: actions/setup-node@v3
+          with:
+              node-version: '18.x'
+        - name: Update Sandbox Repository
+          env:
+              SSH_SANDBOX_DEPLOY_KEY: ${{ secrets.SSH_SANDBOX_DEPLOY_KEY }}
+              SANDBOX_REPOSITORY: ${{ secrets.SANDBOX_REPOSITORY }}
+          run: make update-sandbox
```

Comment:
<!-- Nitpicks are trivial preference-based requests. These should be non-blocking by nature.  -->
**nitpick (non-blocking):** ‏Not sure that we will use this Makefile command manually. We may could directly run the script here and remove the new Makefile command.

Reply:
Useful for manual testing